packages/core/lib/core.js

```diff
@@ -73,6 +74,7 @@ module.exports = {
         // Force the admin & port
         ...(appEntry.admin ? { adminUI: appEntry.admin } : {}),
         port,
+        distDir: distDir || appEntry.distDir || 'dist',
```

Comment:
`'dist'` here should probably be a default like the `entryFile`/`port` above:

```suggestion
        distDir: distDir || appEntry.distDir || DEFAULT_DISTDIR,
```

That can then be exported above, and imported into the `build.js` command to be used there:

https://github.com/keystonejs/keystone-5/pull/1088/files#diff-e3b5d2c74bc1774830696f662545611cR24